lang/fpc/Portfile

```diff
@@ -4,33 +4,46 @@ PortSystem          1.0
 
 name                fpc
 version             3.2.2
-categories          lang
-platforms           darwin
-license             GPL-2 LGPL-2
+categories          lang pascal
```

Comment:
macports has no pascal category

Reply:
@kencu Now it will have :)

We have an approval for new subcategories:
https://trac.macports.org/ticket/67685

Reply:
oh, I see how yoy’re interpreting Josh’s comment :)

OK, well might as well create it and put this lonely soldier in it then, so it exists, and see what he says.

Reply:
@kencu Ken, seriously, what is wrong with it? I can remove the category, no drama, but whom does is help? How is it self-evident that `fpc` is a Pascal compiler? Try [searching](https://ports.macports.org/search/?q=pascal&name=on) for `pascal` in ports, you will not find it now. Also, we do have subcategories for several languages already.

> put this lonely soldier in it then

Well, if you actually check, there is also another Pascal compiler in Macports.